xml/System.IO/FileSystemAclExtensions.xml

```diff
@@ -104,6 +135,37 @@ This extension method was added to .NET Core to bring the functionality that was
 
 This extension method was added to .NET Core to bring the functionality that was provided by the `System.IO.FileStream.#ctor(System.String,System.IO.FileMode,System.Security.AccessControl.FileSystemRights,System.IO.FileShare,System.Int32,System.IO.FileOptions,System.Security.AccessControl.FileSecurity)` .NET Framework constructor.
 
+## Example
+
+The following code example creates a new text file inside the user's temporary folder with the specified file security attributes:
+
+```csharp
+using System.IO;
+using System.Security.AccessControl;
+using System.Security.Principal;
+
+namespace ConsoleApp
+{
+    class Program
+    {
+        static void Main()
+        {
+            FileSecurity security = new FileSecurity();
+
+            SecurityIdentifier identity = new SecurityIdentifier(WellKnownSidType.BuiltinUsersSid, null);
+            FileSystemAccessRule accessRule = new FileSystemAccessRule(identity, FileSystemRights.FullControl, AccessControlType.Allow);
+            security.AddAccessRule(accessRule);
+
+            string path = Path.Combine(Path.GetTempPath(), "fileToCreate.txt");
+
+            FileInfo fileInfo = new FileInfo(path);
+
+            fileInfo.Create(FileMode.Create, FileSystemRights.FullControl, FileShare.Read, 4096 /* default buffer size */, FileOptions.None, security);
```

Comment:
the text `default buffer size` won't be localized. I would just put this in the introduction of the code:

`The following code example creates a new text file (with the default buffer size of 4096) inside the user's temporary folder with the specified file security attributes`

Reply:
```suggestion
            fileInfo.Create(FileMode.Create, FileSystemRights.FullControl, FileShare.Read, 4096, FileOptions.None, security);
```